src/features/document/document_feature.py

```diff
@@ -174,3 +175,18 @@ def remove(address: str, user: User = Depends(auth_w_jwt_or_pat)):
     - str: "OK" (200)
     """
     return remove_use_case(user=user, address=address)
+
+
+@router.get("-existence/{address:path}", operation_id="document_check", response_model=bool, responses=responses)
+@create_response(JSONResponse)
+def check_existence(address: str, user: User = Depends(auth_w_jwt_or_pat)):
+    """Checks if a document exists, given an address.
```

Comment:
```suggestion
    """Checks if an entity exists, given an address.
```

when address is datasource/path/car.engine, the address points to an entity and not a document. This is a definition thing, but all documents are actually entities (but not all entities are documents).

Documents are basically what is stored in mongo db as a separate "entry"